book/030-tutorial-downstream/060-alpha-diversity.md

```diff
@@ -11,3 +11,165 @@ name: tutorial
 default-interface: galaxy-usage
 ---
 ```
+
+## Alpha group significance
+
+First we'll look for general patterns, in particular we may be interested in
+subject and disease state as an influence for alpha diversity.
+
+To start with, we'll examine 'observed features'
+```{usage}
+
+use.action(
+    use.UsageAction('diversity', 'alpha_group_significance'),
+    use.UsageInputs(
+        alpha_diversity=core_metrics_results.observed_features_vector,
+        metadata=sample_metadata),
+    use.UsageOutputNames(visualization='alpha_group_sig_obs_feats')
+)
+```
+
+The first thing to notice is the high variability in individual alpha diversity
```

Comment:
I suggest replacing "alpha diversity" and "diversity" with "richness" throughout this document.